jobs/user-vms/admin-exams.hcl

```diff
@@ -0,0 +1,95 @@
+job "admin-exams" {
+  datacenters = ["aperture"]
+
+  group "ayden-vm" {
+
+    network {
+      mode = "host"
+    }
+
+    service {
+      name = "ayden-vm"
+    }
+
+    task "ayden-vm" {
+      constraint {
+        attribute = "${attr.unique.hostname}"
+        value = "chell" # task must be scheduled on a host with the bridge device configured
+      }
+
+      resources {
+        cpu = 12000
+        memory = 4096
+      }
+
+      artifact {
+        source = "http://136.206.16.5:8000/base-images/debian-12-genericcloud-amd64-30G.qcow2"
+        destination = "local/ayden-vm.qcow2"
+        mode = "file"
+      }
+
+      driver = "qemu"
+
+      config {
+        image_path = "local/ayden-vm.qcow2"
+        accelerator = "kvm"
+        drive_interface = "virtio"
+
+        args = [
+          "-netdev",
+          "bridge,id=hn0",
+          "-device",
+          "virtio-net-pci,netdev=hn0,id=nic1,mac=52:54:84:ba:49:20", # mac address must be unique or else you will regret it
```

Comment:
Add this comment to the README in `jobs/user-vms` too please, or create a new PR with an update for `gh.com/redbrick/docs`.

Reply:
Added the note there in the README. Will also add this in the docs repo